### ISSUE TITLE
Mark messages as read when moving to spam

### ISSUE THREAD

**Issue by steelman** [k9mail/k-9#5767]
**Problem**
Most of the time I move messages to spam without opening them. The message list view is enough. So I mark messages and choose Spam from the menu when messages are still marked as unread. Later, when I open the spam folder to see if there is any new ham,  those messages are still marked as unread and unnecessarily make searching for spam more timeconsuming (I've got one account with a poor filter and a lot of false negatives).

**Solution**
When users choose Spam action, mark messages as read (if users configure so) when (before or after, whatever is more convenient to code for IMAP) moving to spam folder.

**Comment by cketti** [comment]
I don't think this needs to be a setting. I say we mark the message as read when using the 'spam' action. If the user wants to keep the unread state, they can use the 'move' action instead.
